src/styles/reset.scss

```diff
@@ -0,0 +1,7 @@
+body {
+  margin: 0;
+  font-family: $main-font;
```

Comment:
The variable `$main-font` is used here, but it's not defined in this file. Ensure that it is defined in a variables file or imported correctly if it's defined elsewhere.